src/main/java/com/jalasoft/jfc/controller/MetadataConverterController.java

```diff
@@ -37,24 +37,24 @@
 import java.io.IOException;
 
 /**
- * Extracts metadata from a uploaded file.
+ * Extracts metadata of an uploaded file.
  *
  * @version 0.1 08 Jan 2020.
  *
  * @author Juan Martinez.
  */
-
 @RestController
 public class MetadataConverterController {
+
     /**
      * Generates metadata from multipart file.
      * @param file is multipart value.
-     * @return Response
-     * @throws Md5Exception
+     * @param request is client request value.
+     * @return ResponseEntity<Response> for status code.
```

Comment:
It would be ".....for code status"

Reply:
this is correct, becuase it is the response "status code 404" or something like that.